build.sh

```diff
@@ -92,4 +92,9 @@ scripts/feeds install -d y $OTB_PKGS
 cp .config.keep .config
 
 make defconfig
-make "$@"
+set +e
+if ! make "$@" 
+then
+	make "$@" -j1 V=s 2>&1 | tee error.log
+	exit ${PIPESTATUS[0]}
```

Comment:
`PIPESTATUS` is not POSIX,  anyway `exit 1` should be enough ?

Reply:
Also, the current coding style is to put the `then` in the same line as the `if`.